docs/src/pages/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Introduction
+weight: -100
+---
+
+The [HealthCare.gov](https://www.healthcare.gov/) design system is a set of open-source design and front-end development resources for creating usable, accessible (e.g., WCAG/Section 508 compliant), responsive, and consistent websites and web-based solutions.
+
+The Healthcare.gov design system is built on the CMS Design System and extends it to support additional CSS and React components.
+
+The design system is currently being applied to some pages on HealthCare.gov and project teams are actively working to apply the design system consistently across all of HealthCare.gov. 
```

Comment:
I like this section, but maybe we should put it last

Reply:
I'm also curious to what Kristen thinks. There are product using the design system that arent public facing or have these easily accessible links. I also think that we want other products to use the design system even if they arent using it as a dependency or with React.

Reply:
Kristen and Mel helped craft this content in a confluence doc prior to putting in this PR - I could see that content moving to the bottom though

Reply:
I'm all for putting the list of applications at the bottom, good call. Please make sure to point out these are _a few_, not all of the applications currently using HC.gov DS.